inventory.gd

```diff
@@ -0,0 +1,17 @@
+class_name Inv
```

Comment:
Rename to `Inventory`? It doesn't hurt to be more verbose :) InvSlot and similar is fine but I would also consider renaming that to InventorySlot